x-pack/test/functional_with_es_ssl/apps/cases/group1/create_case_form.ts

```diff
@@ -66,6 +66,39 @@ export default ({ getService, getPageObject }: FtrProviderContext) => {
       expect(await button.getVisibleText()).equal('Add connector');
     });
 
+    it('displays errors correctly while creating a case', async () => {
+      const caseTitle = Array(161).fill('x').toString();
+      const longTag = Array(256).fill('a').toString();
+      const longCategory = Array(51).fill('x').toString();
+
+      await cases.create.openCreateCasePage();
+      await cases.create.createCase({
+        title: caseTitle,
+        description: '',
```

Comment:
long description check for 30k characters causes e2e tests to hang for a long time.